macros/audit_logging/execution_model.sql

```diff
@@ -23,7 +24,8 @@
         )
 
     values (
-        '{{ invocation_id }}'::uuid,
+        fn_uuid()::text,
```

Comment:
cast to varchar not text?

Reply:
text is varchar? [see here](https://docs.aws.amazon.com/redshift/latest/dg/c_Supported_data_types.html)